package.json

```diff
@@ -23,9 +23,8 @@
     "@ionic-native/core": "^5.0.0",
     "@ionic-native/splash-screen": "^5.0.0",
     "@ionic-native/status-bar": "^5.0.0",
-    "@ionic/angular": "^4.11.7",
+    "@ionic/angular": "^4.7.1",
     "cordova-android": "8.1.0",
-    "cordova-custom-config": "^5.1.0",
```

Comment:
What there a reason for this revert? I am ok if we are hard coding permission in `AndroidManifest.xml`

Reply:
The plugin threw errors because of the custom configs in the config.xml file and prevented building.